de.fu_berlin.inf.dpp.server/src/de/fu_berlin/inf/dpp/server/ServerLifecycle.java

```diff
@@ -0,0 +1,78 @@
+package de.fu_berlin.inf.dpp.server;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.List;
+
+import org.apache.log4j.Logger;
+
+import de.fu_berlin.inf.dpp.account.XMPPAccount;
+import de.fu_berlin.inf.dpp.account.XMPPAccountStore;
+import de.fu_berlin.inf.dpp.communication.connection.ConnectionHandler;
+import de.fu_berlin.inf.dpp.context.AbstractContextLifecycle;
+import de.fu_berlin.inf.dpp.context.ContainerContext;
+import de.fu_berlin.inf.dpp.context.IContextFactory;
+import de.fu_berlin.inf.dpp.filesystem.IProject;
+import de.fu_berlin.inf.dpp.filesystem.IResource;
+import de.fu_berlin.inf.dpp.net.xmpp.JID;
+import de.fu_berlin.inf.dpp.session.ISarosSessionManager;
+import de.fu_berlin.inf.dpp.session.SessionEndReason;
+
+public class ServerLifecycle extends AbstractContextLifecycle {
+
+    private static final Logger log = Logger.getLogger(ServerLifecycle.class);
+    
+    @Override
+    protected Collection<IContextFactory> additionalContextFactories() {
+        List<IContextFactory> factories = new ArrayList<IContextFactory>();
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Replace the type specification in this constructor call with the diamond operator ("<>"). ([squid:S2293](https://sonarcloud.io/organizations/saros/rules#rule_key=squid%3AS2293))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=saros_complete&pullRequest=281&issues=AWb__iXssSodgw_KF5fR&open=AWb__iXssSodgw_KF5fR)

[ISSUE_KEY]: AWb__iXssSodgw_KF5fR